include/open62541/server_pubsub.h

```diff
@@ -573,6 +575,9 @@ UA_EXPORT UA_StatusCode UA_THREADSAFE
 UA_Server_unfreezeWriterGroupConfiguration(UA_Server *server, const UA_NodeId writerGroup);
 
 UA_EXPORT UA_StatusCode UA_THREADSAFE
+UA_Server_enableWriterGroup(UA_Server *server, const UA_NodeId writerGroup);
```

Comment:
Explain the difference between *enable* and *setOperational*.

We currently have both and redundancy leads to confusion.

Reply:
The _enable_ API pushes the component state to Pre-Operational state if the parent of the Component is Operational and wait for trigger to Operational state. In the current implementation, there is no internal component for Reader/Writer Group and for DataSetReader which can push the state to Operational this needs to be done by the application explicitly using the _setOperational_.

in future the _setOperational_ can be removed in favour of a mechanism/internal pubsub component which pushes the state to Operational.

Reply:
My question was:

We have both setOperational and enable as separate API methods.
What is the difference between them?

It seems to me that “enable” is redundant and will essentially do the same as setOperational (which also drops down to pre-operational if required).